layout/_partial/sidebar-navigation.ejs

```diff
@@ -14,44 +14,38 @@
     </li>
 
     <!-- Homepage -->
-    <li id="sidebar-first-li">
-        <a href="<%= config.root %>" target="_self">
-            <i class="material-icons sidebar-material-icons">home</i>
-            <%= __('sidebar.homepage') %>
-        </a>
-    </li>
-
-    <!-- I'm Feeling Lucky -->
-    <!--
-    <li class="dropdown">
-        <a href="" target="_self">
-            <i class="material-icons sidebar-material-icons">explore</i>
-             <%= __("sidebar.imlucky") %>
-        </a>
-    </li>
-    -->
+    <% if(theme.sidebar.homepage === true) { %>
+            <li id="sidebar-first-li">
+                <a href="<%= config.root %>" target="_self">
+                    <i class="material-icons sidebar-material-icons"><%= theme.sidebar.homepage.icon %></i>
+                <%= __('sidebar.homepage') %>
+            </a>
+        </li>
+    <% } %>
 
     <!-- Archives  -->
-    <li class="dropdown">
-        <a href="#" class="ripple-effect dropdown-toggle" data-toggle="dropdown">
-            <i class="material-icons sidebar-material-icons">inbox</i>
-             <%= __('sidebar.archive') %>
-            <b class="caret"></b>
-        </a>
-        <ul class="dropdown-menu">
+    <% if(theme.sidebar.archives === true) { %>
+        <li class="dropdown">
+            <a href="#" class="ripple-effect dropdown-toggle" data-toggle="dropdown">
+                <i class="material-icons sidebar-material-icons"><%= theme.sidebar.archive.icon %></i>
+                    <%= __('sidebar.archive') %>
+                <b class="caret"></b>
+            </a>
+            <ul class="dropdown-menu">
             <li>
-            <%- list_archives({
-                show_count: true,
-                amount: 5,
-                class: 'sidebar_archives',
-                style: 'none',
-                separator: '</li><li>'
-            }) %>
-        </ul>
-    </li>
+                <%- list_archives({
+                    show_count: true,
+                    amount: 5,
+                    class: 'sidebar_archives',
+                    style: 'none',
+                    separator: '</li><li>'
+                }) %>
+            </ul>
+        </li>
+    <% } %>
 
     <!-- Categories  -->
-    <% if(theme.sidebar_categories === true) { %>
+    <% if(theme.sidebar.categories === true) { %>
         <li class="dropdown">
             <a href="#" class="ripple-effect dropdown-toggle" data-toggle="dropdown">
                 <i class="material-icons sidebar-material-icons">chrome_reader_mode</i>
```

Comment:
- You seem to have forgotten to use the icon from the configuration like `<%= theme.sidebar.categories.icon %>`
- And then surround this line with the following condition: `if(theme.sidebar.categories.icon){`